components/fxa-client/src/send_tab.rs

```diff
@@ -38,16 +40,21 @@ impl FirefoxAccount {
     }
 
     /// Send a single tab to another device designated by its device ID.
+    /// XXX - this should change to taking a Vec of device IDs, because
+    /// collecting telemetry relies on a multi-device send happening in a single
+    /// operation.
```

Comment:
I agree, but also, that's a pretty significant breaking change. It makes me wonder if we should move to supporting a `Vec` of tab data as well for the general case! But then again, desktop doesn't support that and our telemetry doesn't really make that distinction either.

Reply:
Yeah - we could just deprecate. But we'll need this for desktop assuming we want to keep the `flowID` semantics. OTOH, it's not clear that we do :)

Reply:
I've updated the comment but otherwise haven't moved towards actually doing this.